index.js

```diff
@@ -102,6 +102,20 @@ module.exports.lintFiles = (patterns, opts) => {
 			});
 		}
 
+		const isTSFile = ext => ext === '.ts' || ext === '.tsx';
+
+		if (paths.filter(filePath => isTSFile(path.extname(filePath))).length > 0) {
```

Comment:
Instead of `filter` + `.length > 0` you'll get better performances with `find` as it will stop on the first item found:
```js
if (paths.find(filePath => isTSFile(path.extname(filePath))))
```